### ISSUE TITLE
Modify the properties sub-package for the implementation of thermochemical properties

### ISSUE THREAD

**Issue by panDanieleN** [newcleo-dev-team/lbh15#28]
For the sake of clarity, modify the `properties` sub-package structure the handle the large amount of new thermochemical properties.
The sub-package shall look like:

```

properties/
- __init__.py
- bismuth_properties.py
- lbe_properties.py
- lead_properties.py
- bismuth_thermochemicals/
  - __init__.py
  - diffusivity_in_bismuth.py
  - solubility_in_bismuth.py
  - bismuth_thermochemical_properties.py
- lbe_thermochemicals/
  - __init__.py
  - diffusivity_in_lbe.py
  - solubility_in_lbe.py
  - lbe_thermochemical_properties.py
- lead_thermochemicals/
  - __init__.py
  - diffusivity_in_lead.py
  - solubility_in_lead.py
  - lead_thermochemical_properties.py
```

The main `__init__.py` of the package must include all the modules defined in  the `properties` sub-package properly.

In addition, update the code documentation to properly read the new sub-package docstrings.

**Comment by klucie** [comment]
Modification of test_lbh15_bounds.py and properties_bounds.json according to the new organization of the properties sub-package and the new correlations implemented.

branch: modif_thermochemical
commit: 6a03a6e70817f0127fa0da9bd8bc69f1bf855e27

**Comment by klucie** [comment]
Modification of test_lbh15_fromX.py so it tests all the new correlations, in addition to the chapter 2 correlations. 

branch: modif_thermochemical
commit: 80454bc

**Comment by klucie** [comment]
Adding a lbe_oxygen_limits.py file and a lead_oxygen_limits.py file. Final structure should looks like that:  

properties/
- __init__.py
- bismuth_properties.py
- lbe_properties.py
- lead_properties.py
- bismuth_thermochemicals/
  - __init__.py
  - diffusivity_in_bismuth.py
  - solubility_in_bismuth.py
  - bismuth_thermochemical_properties.py
- lbe_thermochemicals/
  - __init__.py
  - diffusivity_in_lbe.py
  - solubility_in_lbe.py
  - lbe_thermochemical_properties.py
  - lbe_oxygen_limits.py
- lead_thermochemicals/
  - __init__.py
  - diffusivity_in_lead.py
  - solubility_in_lead.py
  - lead_thermochemical_properties.py
  - lead_oxygen_limits.py

**Comment by panDanieleN** [comment]
Merged into master.

Documentation update addressed in #38
